ls1intum/Artemis
# [bugfix] fixed button row for programming exercises

### Checklist

### Motivation and Context


Due to a merge conflict some buttons in the exercise overview component were not correct. I fixed them in this PR.

![image](https://user-images.githubusercontent.com/33342534/108989188-9e1f7b00-7695-11eb-9994-c581f5b91938.png)
--->
![image](https://user-images.githubusercontent.com/33342534/108989232-ab3c6a00-7695-11eb-9ff0-4ba11df8d881.png)

### Steps for Testing


Go to course Management -> "Exercises" of an arbitrary course with programming exercises. The programming exercise's buttons should not display View, and not display the grading button twice.
